### ISSUE TITLE
Alias to user

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/server#26566]
- [ ] Android @tobiasKaminsky @mahibi 
- [ ] Desktop @nextcloud/desktop 
- [ ] iOS @Ivansss 
- [ ] Server @nextcloud/server-triage @nickvergessen 

iOS already has the possibility to set a more descriptive name to the account.
If we do this on all clients, it should be also on server, and part of the profile page, or?

**Comment by blizzz** [comment]
--verbose

What is it an alias for?

**Comment by nickvergessen** [comment]
It was added to the iOS files app recently and now other clients are being asked by the community to implement it. So this issue is to coordinate the feature: (sorry for german screenshots)
![alias-setting](https://user-images.githubusercontent.com/213943/114847211-c0a05b80-9ddd-11eb-9347-86292d7d0618.jpg)

![alias-showing](https://user-images.githubusercontent.com/213943/114847258-cd24b400-9ddd-11eb-8367-7e8ccdd4b2cb.jpg)

**Comment by blizzz** [comment]
So, this is just another user-defined name for the account? i.e. "Private" instead of "Nextcloud"?

**Comment by rullzer** [comment]
I don't think this is anything server related. Giving your accounts there an alias seems only confusing to me

**Comment by blizzz** [comment]
> I don't think this is anything server related. Giving your accounts there an alias seems only confusing to me

Well, it would be a short cut to name it once and all aware clients would use it automatically once authenticated. But whether this is really worth the effort… This would even imply you want the same alias everywhere. Otherwise you need options to override… Let's just the let clients the clients offer to set an alias.

**Comment by elsiehupp** [comment]
If the Alias is Apple-specific, it would make sense to sync the setting via iCloud. At the moment, that would only cover iOS and iPadOS, but it could be expanded to macOS if Nextcloud Desktop were ever to ship via the Mac App Store (which would confer other advantages and tradeoffs). That said, the macOS Finder is still quite different form the iOS and iPadOS Files app, so it might be of dubious utility.

**Comment by tobiasKaminsky** [comment]
@marinofaggiana can you share your idea behind this feature?

**Comment by iGeorgeX** [comment]
I was wondering if creating alias names is now part of the desktop client. I couldn't find anything on this except this rather old question.
